Apromore-Core-Components/Apromore-Manager/src/main/resources/db/migration/sql/db-data-h2.sql

```diff
@@ -0,0 +1,119 @@
+
+INSERT INTO native_type VALUES (1,'EPML 2.0','epml');
+INSERT INTO native_type VALUES (2,'XPDL 2.2','xpdl');
+INSERT INTO native_type VALUES (3,'PNML 1.3.2', 'pnml');
+INSERT INTO native_type VALUES (4,'YAWL 2.2', 'yawl');
+INSERT INTO native_type VALUES (5,'BPMN 2.0', 'bpmn');
+INSERT INTO native_type VALUES (6,'AML fragment', 'aml');
+
+
+INSERT INTO role VALUES (1,'80da507e-cdd7-40f4-a9f8-b2d2edb12856','ROLE_ADMIN','Ultimate power to administer other users');
+INSERT INTO role VALUES (2,'0ecd70b4-a204-41cd-a246-e3fcef88f6fe','ROLE_USER','Allowed to log in');
+INSERT INTO role VALUES (3,'72503ce0-d7cd-47b3-a33c-1b741d7599a1','ROLE_MANAGER','Allowed to administer folders, models and event logs');
+INSERT INTO role VALUES (4,'e0ae8c02-bfe0-11ea-992a-17a70e2c7d0f','ROLE_ANALYST','Allowed access to all tools except predictive monitoring training');
+INSERT INTO role VALUES (5,'43fbd3c8-bfe1-11ea-9694-abeaa8ed1c0f','ROLE_OBSERVER','Allowed access to all tools in read-only mode');
+INSERT INTO role VALUES (6,'4ff46cbc-bfe1-11ea-abec-aba9c1ea1178','ROLE_DESIGNER','Allowed access to the BPMN editor');
+INSERT INTO role VALUES (7,'596fcfd4-bfe1-11ea-8564-4f9db22c941e','ROLE_DATA_SCIENTIST','Allowed access to all tools including predictive monitoring training');
+INSERT INTO role VALUES (8,'64e734a6-bfe1-11ea-b4cf-2bed596c2920','ROLE_OPERATIONS','Allowed access to the predictive monitoring dashboard (runtime)');
+
+
+INSERT INTO user VALUES (1,'75f4a46a-bd32-4fbb-ba7a-c50d06414fac','james','2012-05-23 11:52:48','Cameron','James','2012-05-23 11:52:48',1,NULL,NULL,NULL,NULL,NULL);
```

Comment:
User table: data should be cleaned up as they are real, sensitive but unused.

Reply:
Commented as above

Reply:
As you're doing this migration to liquibase, it's a legacy issue to get rid of. Pls contact with Marcello as he knows what records to remove.